tests/cephadm/test_config.py

```diff
@@ -0,0 +1,60 @@
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def set_multiple_public_network(cls):
+    """
+    To set multiple public network to ceph.
+    Args:
+        cls: cephadm instance object
+
+    Returns:
+        False -> Fail, True -> Pass
```

Comment:
Could you please add more description as to, when you say fail what failed (setting public network failed or any other validation post that)? similarly for pass.